dialog/confirm_basic_auth.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @copyright Robin Jadoul
+ *
+ * Copyright (C) 2018  Robin Jadoul
+ * This file is part of Keywi.
+ * Keywi is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Keywi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Keywi.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const generateButtonRow = function (name, login) {
```

Comment:
Could or should this be extracted for reuse with the ordinary `getGUILogins`?

Reply:
Yes, but I would prefer to do this with #105